packages/thirdweb/src/utils/bytecode/selectors.ts

```diff
@@ -0,0 +1,33 @@
+import { selectorsFromBytecode } from "@shazow/whatsabi";
+import type { ThirdwebContract } from "../../contract/contract.js";
+import { resolveImplementation } from "../../utils/bytecode/resolveImplementation.js";
+import { LruMap } from "../../utils/caching/lru.js";
+
+const SelectorCache = new LruMap<string[]>(8192);
```

Comment:
8k contracts in cache? seems like a lot no?

Reply:
it's really just "don't get TOO big" - the other option was just a normal map, there's no reason to ever do it again once we've done it once